thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/wrapper/AnomalyDetectorWrapper.java

```diff
@@ -48,12 +48,14 @@
 import org.apache.pinot.thirdeye.util.ThirdEyeUtils;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
+import org.joda.time.Duration;
 import org.joda.time.Interval;
 import org.joda.time.Period;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import static org.apache.pinot.thirdeye.dataframe.util.DataFrameUtils.*;
+import static org.apache.pinot.thirdeye.detection.DetectionUtils.*;
```

Comment:
This is not LinkedIn coding convention.
https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Java+Coding+Conventions#JavaCodingConventions-2.5.1AvoidStarImports

Reply:
Thanks for pointing out. But this is static utility classes, so I assume it's ok as said in the doc?

Reply:
Cleaned up